tools/mo/openvino/tools/mo/front/output_cut.py

```diff
@@ -35,9 +36,15 @@ def find_and_replace_pattern(self, graph: Graph):
                     if fw_info_value:
                         fw_info = fw_info_value
                         break
+            if fw_info is not None and in_node is not None:            
+                for out_idx in in_node.out_nodes():
+                    out_node = in_node.out_node(out_idx)
+                    out_nodes_ids[out_idx] = out_node.id
+            
             graph.erase_node(node)
 
             if fw_info is not None and in_node is not None:
                 for out_idx in in_node.out_nodes():
-                    set_edge_attribute_between_nodes(in_node, in_node.out_node(out_idx),
+                    if node.id == out_nodes_ids[out_idx]:
```

Comment:
should not we save `node.id` value in `node_id` due to `graph.erase_node(node)` call before?

Reply:
Ok, thanks!